float32 transBearing
float32 rangeToTarget
float32 heading